packages/cli/src/commands/init/init.ts

```diff
@@ -58,6 +60,30 @@ interface TemplateOptions {
   installCocoaPods?: string | boolean;
 }
 
+const YARN_VERSION = '3.6.4';
+
+const bumpYarnVersion = async (silent: boolean, root: string) => {
+  try {
+    let yarnVersion = semver.parse(getYarnVersionIfAvailable());
+    if (yarnVersion && semver.major(yarnVersion) === 1) {
+      await executeCommand('yarn', ['set', 'version', YARN_VERSION], {
+        root,
+        silent,
+      });
+
+      // React Native doesn't support PnP, so we need to set nodeLinker to node-modules. Read more here: https://github.com/react-native-community/cli/issues/27#issuecomment-1772626767
+
+      await executeCommand(
+        'yarn',
+        ['config', 'set', 'nodeLinker', 'node-modules'],
```

Comment:
Another config option we may want to set by default is `nmHoistingLimits: 'workspaces'`

This is not necessary in regular projects - but maybe necessary if the project is converted to a monorepo later.

Reply:
✅